sdx_pushgateway_sg.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_security_group" "sdx_pushgateway" {
-  count       = local.is_management_env ? 0 : 1
+  count       = 1
```

Comment:
why do the count? shouldn't it run just once anyway? 

```suggestion
```

Reply:
the count has to be there because the resources have been deployed to non-mgmt environments already. Without it, it won't work for all environments in this PR. That count can be removed when all environments are the same.